package.json

```diff
@@ -36,5 +36,8 @@
   "engines": {
     "node": ">= 15.13.0",
     "npm": ">= 7.7.6"
+  },
```

Comment:
Hmm, this shouldn't be here

Reply:
(the "@tailwindcss/postcss7-compat") part below

Reply:
I just reverted that commit. I added this because the first failure below was telling me include it:

`Error:   1:16  error  '@tailwindcss/postcss7-compat' should be listed in the project's dependencies. Run 'npm i -S @tailwindcss/postcss7-compat' to add it  import/no-extraneous-dependencies`